kaamaurice/OpenPype
# Fix: preview range cuts review playblasts

# Description
Prevent preview range from affecting review playblast frame range by deactivating preview range during extraction.

# Testing
- Create or load an animation workfile with at least an animation instance and a review instance
- In the dopesheet, turn on preview range and change the frame range to something shorter than the expected playblast frame range (So we're in a case that would affect the playblast)
- Publish
- Go to kitsu and check that all frames have been rendered in the playblast (The animation should "freeze" outside of the preview range if it doesn't work)
